packages/viewer/src/lib/types/flags.ts

```diff
@@ -28,8 +28,14 @@ type Flag =
    * are larger than the supported maximum image size (1080p). By default
    * this option is disabled, and images will be scaled up to match the host.
    */
-  | 'letterboxFrames';
+  | 'letterboxFrames'
 
+  /**
+   * Enables or disables the use of the `grpc.WebsocketTransport` for scene
+   * tree subscriptions. By default, this transport will be used, and
+   * subscription messages will be sent over WebSocket.
+   */
+  | 'useSubscriptionWebSocketTransport';
```

Comment:
Let's generalize to something like `grpcUseStreamingWebSocketTransport`. May need to be used for other streaming endpoints.